src/svnFinder.ts

```diff
@@ -112,9 +112,11 @@ export class SvnFinder {
 
   public checkSvnVersion(svn: ISvn): Promise<ISvn> {
     return new Promise<ISvn>((c, e) => {
-      if (!semver.valid(svn.version)) {
+      // fix compatibility with SlickSVN (like 1.6.17-SlikSvn-tag-1.6.17@1130898-X64)
+      const version = svn.version.replace(/^(\d+\.\d+\.\d+).*/, "$1");
```

Comment:
whats the `.*` needed for? `^(\d+\.\d+\.\d+)` gets the version alone

Reply:
Fast version instead of match or exec